gui/widget.cpp

```diff
@@ -54,6 +54,10 @@ void Widget::init() {
 	_next = _boss->_firstWidget;
 	_boss->_firstWidget = this;
 	_needsRedraw = true;
+
+	if (TransMan.getCurrentLanguage() == "C" && false) {		// GUI TODO: Change this and GuiManager::GuiManager() to use RTL.
```

Comment:
I do not see you taking `nortl` XML flag into account

Reply:
This is fixed now, every widget gets their internal flags from the function in `gui/object.cpp`, in `GuiObject::reflowLayout()`